aten/src/ATen/native/cpu/ReducedPrecisionFloatGemvFastPathKernel.cpp

```diff
@@ -104,7 +104,8 @@ float fp16_dot_with_fp16_arith(const Half* x, const Half* a, int len) {
 // Rather than unrolling to process multiple rows (transposed columns)
 // of matrix A at once as done in fp16_gemv_trans_fp16_arith, unroll
 // along an individual dot product.
-static void fp16_gemv_trans_fp16_arith_by_dot_products(const int m, const int n, const Half* a, const int lda, const Half *x, const float beta, Half* y, int incy) {
+// NB: lda must be long, otherwise it can cause int32 overflow
```

Comment:
long can be compiler dependent, might be better to use size_t or uint64t?

Reply:
Let me change it to int64_t, and probably file a followup issue to figure out why all those types are signed integers to begin with (but I guess it's implicitly inherited from BLAS, I don't know if Fortran have an unsigned types to begin with :) )